container-registry-configuration/README.md

```diff
@@ -0,0 +1,189 @@
+# Container Registry Configuration
+
+The purpose of this RFC is to specify how we configure Kubernetes clusters for container registries.
+
+## Problem Statement
+
+Docker Hub has [download rate limit](https://docs.docker.com/docker-hub/download-rate-limit/) and it is pretty strict. To use Docker Hub without authentication is not a livable option for us. We started to hit the limit in big CAPI clusters during upgrades.
+
+## Current status
+
+- We use images from different registries. Here is the summary of registry usage of a test MC.
+```
+explicit docker.io  40
+implicit docker.io  2
+explicit quay.io    26
+explicit k8s.gcr.io 12
```

Comment:
is this per node basis or whole cluster? (some images will be the same)

Reply:
It is for the whole MC cluster. Images are unique.

Reply:
as a driveby note, it'd be good to retag any images from `k8s.gcr.io` to our own registries

Reply:
> as a driveby note, it'd be good to retag any images from k8s.gcr.io to our own registries

also because they won't work in China, we had to do that already

Reply:
Just a note: `k8s.gcr.io` is the old kubernetes registry and shouldn't be used anymore anyway.

Reply:
The new one is `registry.k8s.io`